pkg/controller/configmap_watcher.go

```diff
@@ -54,6 +54,7 @@ func NewConfigMapWatcher(cfg *rest.Config) (ConfigMapWatcher, error) {
 	return ConfigMapWatcher{
 		clientset: clientset,
 		logger:    log.Log.WithName("ConfigMapWatcher"),
+		ch:        make(chan config.Config),
```

Comment:
Another approach could be to just have it for an empty struct{} to notify about the event and then obtain the config on the listener side.

Reply:
I missed this comment.

Now this structure exists as a field of the Validation Engine structure. This approach would work by exporting the configuration within the structure and making the Validation Engine logic to retrieve it.

Is there any security concern or convention for this to change to an empty struct? (btw I think this was the previous approach when the watcher was coded and merged)

Reply:
Not sure about security concern. My experience is with channels with `struct{}` or perhaps string - i.e keep them lightweight. Let's keep this as it is and focus on more important problems here. We can always update in the future.

Reply:
This will be achieved in this task https://issues.redhat.com/browse/DVO-142